packages/phishing-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/phishing-controller",
-  "version": "10.0.0",
+  "version": "10.1.0",
```

Comment:
FYI, for this repo we bump versions of packages in "release PRs", so we shouldn't do this here yet:

``` suggestion
  "version": "10.0.0",
```